### PR TITLE
Removed the "advanced user" statement

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/465)

### CODE REVIEW

File: content/docs/pipelines/enable-gpu-and-tpu.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = "Enable GPU and TPU for Kubeflow Pipelines on Google Kubernetes En
 weight = 6 
 +++
 
-This page is for advanced users. It describes how to enable GPU or TPU for the pipeline on GKE by using the Pipelines 
+It describes how to enable GPU or TPU for the pipeline on GKE by using the Pipelines
```
Comment (sarahmaddox) [inline content/docs/pipelines/enable-gpu-and-tpu.md]: s/It/This page

Comment (hongye-sun) [reply content/docs/pipelines/enable-gpu-and-tpu.md]: Done

**Pull request merged**